plugin/ui.tsx

```diff
@@ -877,6 +877,11 @@ class App extends SafeComponent {
                           defaultMessage="autolayout"
                         />
                       </a>
+                      <br />
+                      <FormattedMessage
+                        id="importLayerHelp2"
+                        defaultMessage="Vectors should be rasterized"
+                      />
```

Comment:
It would be great if this can link to docs as well, to help folks understand what raserize means. Maybe we can use this link: https://github.com/BuilderIO/figma-html/#auto-layout-vectors

Reply:
Added link